code/game/objects/items/cards_ids.dm

```diff
@@ -698,6 +701,44 @@ update_label("John Doe", "Clowny")
 	hud_state = JOB_HUD_RAWCARGO
 	access = list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MECH_MINING, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM)
 
+/obj/item/card/id/paper
+	name = "paper nametag"
+	desc = "Some spare papers taped into a vague card shape, with a name scribbled on it. Seems trustworthy."
+	icon_state = "paper"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 50, "stamina" = 0)
+	resistance_flags = null  // removes all resistance because its a piece of paper
+	access = list()
+	assignment = "Unknown"
+	hud_state = JOB_HUD_PAPER
+	electric = FALSE
+
+/obj/item/card/id/paper/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/pen))
+		var/target_name = stripped_input(user, "What name would you like to write onto the card?", "Written name:", registered_name || "John Doe", MAX_MESSAGE_LEN)
+		registered_name = target_name || registered_name  // in case they hit cancel
+		assignment = "Unknown"
+		to_chat(user, "<span class='notice'>You scribble the name [target_name] onto the slip.</span>")
+		update_label()
+
+/obj/item/card/id/paper/alt_click_can_use_id(mob/living/user)
+	to_chat(user, "<span class='warning'>There's no money circuitry in here!</span>")
+
+/obj/item/card/id/paper/insert_money(obj/item/I, mob/user, physical_currency)
+	to_chat(user, "<span class='warning'>You can't insert money into a slip!</span>")  // not sure if this is triggerable but just as a safeclip
+
+/obj/item/card/id/paper/GetAccess()
+	return list()
+
+/obj/item/card/id/paper/update_label(newname, newjob)
+	if(newname || newjob)
+		name = "[(!newname)	? "paper slip identifier": "[newname]'s paper slip"]"
+		return
+
+	name = "[(!registered_name)	? "paper slip identifier": "[registered_name]'s paper slip"]"
+
+/obj/item/card/id/set_hud_icon_on_spawn(jobname)
```

Comment:
you override the original proc incorrectly.

Reply:
whats the proper way to do it

Reply:
why are you even trying to override it? I see this code has no reason to exist.

Reply:
/obj/item/card/id/**paper**/set_hud_icon_on_spawn

also, just remove the procc